packages/domainproperties/src/components/assay/AssayDesignerPanels.tsx

```diff
@@ -286,7 +285,7 @@ export class AssayDesignerPanels extends React.PureComponent<Props, State> {
                             domain={domain}
                             headerPrefix={protocolModel.name}
                             controlledCollapse={true}
-                            initCollapsed={initCollapsed}
+                            initCollapsed={currentPanelIndex !== (i+1)}
```

Comment:
The logic here is a little different, but I assume that is intentional?

Reply:
Yes